typing/jkind.ml

```diff
@@ -768,16 +768,22 @@ module Layout_and_axes = struct
       | Sufficient_fuel, Sufficient_fuel -> Sufficient_fuel
   end
 
-  (* Normalize the jkind. If mode is Require_best, only jkinds that are best will be used.
-     If mode is Ignore_best, then Not_best will be used. Since Ignore_best can use
-     Not_best, the result is guaranteed to have no with-bounds.
-
-     At each step during normalization, before expanding a type, [map_type_info] is used
-     to map the type-info for the type being expanded. The type can be prevented from
-     being expanded by mapping the relevant axes to an empty set. [map_type_info] is used
-     by sub_jkind_l to remove irrelevant axes. *)
+  (* Normalize the jkind. If mode is Require_best, only jkinds that are best
+     will be used.  If mode is Ignore_best, then Not_best will be used. Since
+     Ignore_best can use Not_best, the result is guaranteed to have no
+     with-bounds.
```

Comment:
```suggestion
  (* Normalize the jkind. If mode is [Require_best], only jkinds that have quality [Best]
     will be used.  If mode is [Ignore_best], then jkinds that have quality [Not_best] will also be used. Since
     [Ignore_best] can use [Not_best] jkinds, the result is guaranteed to have no
     with-bounds.
```